CHANGELOG.md

```diff
@@ -86,13 +79,23 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased 2.x]
```

Comment:
Nit: Update `[Unreleased 2.x]: https://github.com/opensearch-project/OpenSearch/compare/2.5...2.x` to `[Unreleased 2.x]: https://github.com/opensearch-project/OpenSearch/compare/2.7...2.x`

Reply:
Good catch!